app/views/account_home/show.html.erb

```diff
@@ -0,0 +1,82 @@
+
+<div class="govuk-grid-row govuk-main-wrapper">
+  <div class="govuk-grid-column-one-third">
+    <%= render "account-navigation", page_is: yield(:location) %>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <main id="content" role="main">
+      <%= render "user-reminders" %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: yield(:title),
+        heading_level: 1,
+        font_size: "l",
+        margin_bottom: 4,
+      } %>
+      <% if @user["email"] %>
+        <p class="govuk-body accounts-your-account__email">
+          <%= @user["email"] %>
+        </p>
+      <% end %>
+      <% if @user.dig("services", "transition_checker") == "yes" %>
+        <p class="govuk-body">You've used the transition checker, so we should put the non-empty service card here.</p>
+        <div class="accounts-panel">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: t("account.your_account.transition.heading"),
+            heading_level: 2,
+            font_size: "m",
+            margin_bottom: 4,
+          } %>
+
+          <p class="govuk-body"><%= t("account.your_account.transition.description") %></p>
+
+          <p class="govuk-body govuk-!-margin-0">
+            <a href="<%= transition_checker_path %>/saved-results" class="govuk-link" data-module="gem-track-click" data-track-category="account-manage" data-track-action="your-account" data-track-label="see-results">
+              <%= sanitize(t("account.your_account.transition.link1")) %>
+            </a>
+          </p>
+          <p class="govuk-body"><%= t("account.your_account.transition.link1_description") %></p>
+
+          <p class="govuk-body govuk-!-margin-0">
+            <a href="<%= transition_checker_path %>/edit-saved-results" class="govuk-link" data-module="gem-track-click" data-track-category="account-manage" data-track-action="your-account" data-track-label="update-results">
+              <%= t("account.your_account.transition.link2") %>
+            </a>
+          </p>
+          <p class="govuk-body"><%= t("account.your_account.transition.link2_description") %></p>
+        </div>
+      <% elsif @user.dig("services", "transition_checker") == "yes_but_must_reauthenticate" %>
+        <div class="accounts-panel">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: t("account.your_account.account_not_used.heading"),
+            heading_level: 2,
+            font_size: "m",
+            margin_bottom: 4,
+          } %>
+
+          <p class="govuk-body"><%= t("account.your_account.account_not_used.description") %></p>
+
+          <p class="govuk-body"><%= sanitize(t("account.your_account.account_not_used.action", link: link_to(t("account.your_account.account_not_used.link_text"), "#{transition_checker_path.to_s}/edit-saved-results", html_options = { class: "govuk-link"} ))) %></p>
+        </div>
```

Comment:
The brexit checker has been used in this case, but the user will need to do MFA again to access it (as they signed in at a lower level of authentication).

Since we don't have any designs for that yet, and we don't have the ability to sign in at a lower level switched on in production, this can be combined with the case above, eg

```
<% if has_used? "transition_checker" %>
  "used the checker" case goes here
<% else %>
  "account not used" case goes here
<% end %>
```